UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -93,5 +93,30 @@ public async Task WhenRequestUploadFileShareServiceData_ThenReturnsCloudBlockBlo
 
             Assert.AreEqual("http://tempuri.org/blob", result);
         }
+
+        [Test]
+        public void WhenRequestDownloadFileShareServiceFiles_ThenReturnsFileTospecificPath()
+        {
+            var message = new SalesCatalogueServiceResponseQueueMessage() { 
+                BatchId = "63d38bde-5191-4a59-82d5-aa22ca1cc6dc"
+            };
+            var fulfillmentDataResponses = new List<FulfillmentDataResponse>() {
```

Comment:
Assuming we change the spelling of fulfillment to fulfilment in the other areas this variable will need to be updated to match.

Reply:
updated